app/uk/gov/hmrc/vatregisteredcompanies/schedulers/PayloadConversionScheduler.scala

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.vatregisteredcompanies.schedulers
+
+import java.util.concurrent.TimeUnit
+
+import akka.actor.ActorSystem
+import com.google.inject.AbstractModule
+import javax.inject.{Inject, Singleton}
+import play.api.Logger
+import uk.gov.hmrc.vatregisteredcompanies.services.PersistenceService
+
+import scala.concurrent.ExecutionContext
+import scala.concurrent.duration.FiniteDuration
+
+@Singleton
+class PayloadConversionScheduler @Inject()(persistenceService: PersistenceService, actorSystem: ActorSystem)(
+  implicit val ec: ExecutionContext) {
+
+  private val logger = Logger(getClass)
+
+  // TODO make configurable
```

Comment:
inject this

Reply:
move this to constructor paramter - annotate that paramter it with `@Named("interval")`

```
class PayloadConversionScheduler @Inject()(persistenceService: PersistenceService, 
                                           actorSystem: ActorSystem,
                                           @Named("interval") interval: FinateDuration)(
```

Then in module you can write simple provider.

```
class Module extends AbstractModule {

  @Provides
  @Named("interval")
  def interval() = FiniteDuration(30, TimeUnit.SECONDS)
  

  def configure() = {

 }
}
```